ui.go

```diff
@@ -452,7 +452,7 @@ func (win *win) printDir(screen tcell.Screen, dir *dir, selections map[string]in
 
 		tag, ok := tags[path]
 		if ok {
-			if i == dir.pos {
+			if i == dir.pos && gOpts.hasfocus {
 				win.print(screen, lnwidth+1, i, st.Reverse(true), tag)
 			} else {
```

Comment:
I think this can be simpler. `st` is already the style we want, it already has `Revese(true)` set. 

```
			if i == dir.pos {
				win.print(screen, lnwidth+1, i, st, tag)
			} else {
```

That makes it work with my suggestion above. (I may or may not get around to making these suggestions into a pull request on top of your pull request.

Reply:
Here's a diff with these changes + a related feature: https://github.com/ilyagr/lf/compare/13ff4305a352ecff2a47cfb5d9c3f91211e6bbe2...2673fc745806ee2b16885b67f9d8c18819565ccb

Reply:
I now made the "related feature" into a proper pull request: https://github.com/gokcehan/lf/pull/924.